src/helpers/_typography.scss

```diff
@@ -109,6 +110,8 @@
 
     @if $breakpoint == null {
       font-size: $font-size;
+      // sass-lint:disable no-duplicate-properties
```

Comment:
This disables this linter for the rest of the file. If you want to disable it for just the one line, add the comment to the end of the line as per https://github.com/sasstools/sass-lint#disabling-linters-via-source